data/map.txt

```diff
@@ -18785,29 +18710,21 @@ system Perfica
 	object
 		sprite planet/ice5
 		distance 351
-		period 94
+		period 588.173
 	object
 		sprite planet/dust6-b
 		distance 652
-		period 174
+		period 1489.07
 	object
 		sprite planet/cloud6-b
 		distance 962
-		period 348
-	object
-		sprite planet/desert4-b
-		distance 1757.75
-		period 896.982
-	object
-		sprite planet/gas4
-		distance 2712.71
-		period 1719.7
```

Comment:
Is there a reason why these two objects were removed here?

Reply:
This is a brown dwarf system, and I reduced the number of objects in the few systems that had a brown dwarf instead of a star.

Reply:
Alright. Thanks for the explanation.